src/AccessibilityInsights.SetupLibrary/VersionSwitcherWrapper.cs

```diff
@@ -99,8 +99,9 @@ private static bool TryRecursiveCopy(string source, string dest, List<FileStream
                 foreach (string file in Directory.GetFiles(source))
                 {
                     FileInfo fileInfo = new FileInfo(file);
-                    fileInfo.CopyTo(Path.Combine(dest, file), true);
```

Comment:
>file [](start = 55, length = 4)

`Path.Combine` was throwing here because `file` is the fully qualified path instead of just the name of the file. By using `fileInfo.Name`, we get just the file name.